README.md

```diff
@@ -127,6 +127,11 @@ options:
   -s, --stdout          print changed text to stdout. defaults to true when formatting stdin, or to false otherwise
 ```
 
+To ignore the file, you can also add a comment to the top of the file:
+```python
+# autoflake: ignore
```

Comment:
can you make it `skip_file` instead?

It's more explicit. `ignore` can be a bit vague (is it ignoring the next line? the whole file?)

Reply:
Replaced with `skip_file`, thank you!